src/e2e-tests/features/dashboard.feature

```diff
@@ -2,16 +2,131 @@
 Feature: RoboDomo Dashboard page
     As a User
     I want to see Dashboard page
-    So that I can see all Dashboard widgets
+    So that I can see and use all Dashboard widgets
 
-    @S1 @automated
-    Scenario Outline: Dashboard tabs
+    Background:
         Given User navigates to Dashboard page
         Then User is redirected to Dashboard page
         Then Dashboard tab shall be selected
-        When User clicks on <tabName> tab
-        Then <tabName> tab is loaded
+
+    @S1 @automated
+    Scenario: Check Time tile at Theater tab
+        When User clicks on Theater tab
+        Then Theater tab is loaded
+        Then Current date is displayed at Theater/Bedroom tab of Dashboard page
+        And  Current time is displayed at Theater/Bedroom tab of Dashboard page
+        And Sunrise is displayed at Theater/Bedroom tab of Dashboard page
+        And Sunset is displayed at Theater/Bedroom tab of Dashboard page
+
+    @S1 @automated
+    Scenario Outline: Check Location and Wind tile at Theater tab
+        When User clicks on Theater tab
+        Then Theater tab is loaded
+        Then <city> is displayed at Theater/Bedroom tab of Dashboard page
+        And Temperature is displayed at Theater/Bedroom tab of Dashboard page
+        And Wind direction is displayed at Theater/Bedroom tab of Dashboard page
+        Examples:
+            | city      |
+            | San Diego |
+
+    @S1 @automated
+    Scenario: Check Temperature tile and its counter at Theater tab
+        When User clicks on Theater tab
+        Then Theater tab is loaded
+        Then Inside temp parameter is displayed at Theater/Bedroom tab of Dashboard page
+        When User decreases heat at Theater/Bedroom tab of Dashboard page
+        Then Heat is decreased at Theater/Bedroom tab of Dashboard page
+        When User increases heat at Theater/Bedroom tab of Dashboard page
+        Then Heat is increased at Theater/Bedroom tab of Dashboard page
+
+
+  #  @S1 @automated
```

Comment:
if these are not done yet, you can uncomment them, remove @automated and add @pending

Don't let them commented please. thx